content/en/getting_started/synthetics/_index.md

```diff
@@ -0,0 +1,215 @@
+
+To create a browser test:
+
+1. In the Datadog application, hover over **UX Monitoring** in the left hand menu and select *Synthetics Test*. 
+2. In the top right corner, click the *New Test* button. 
+3. Select *Browser Test*. 
+4. Define the configuration of your browser test.
+
+1. **Starting URL**: Set the URL from which your browser test starts the scenario.
+    * Advanced Options (optional): Set custom request headers, cookies, or authenticate through Basic Auth.
+        * Headers: Defined headers override the default browser headers. For example, set the User Agent in the header to [identify Datadog scripts][10].
+        * Authentication: Authenticate through HTTP Basic Authentication with a username and a password.
+        * Cookies: Defined cookies are added to the default browser cookies. Set multiple cookies using the format `<COOKIE_NAME1>=<COOKIE_VALUE1>; <COOKIE_NAME2>=<COOKIE_VALUE2>`.
+
+2. **Name**: Set the name of your browser test.
+3. **Select your tags**: Select the tags attached to your browser test. Use the `<KEY>:<VALUE>` format to filter on a `<VALUE>` for a given `<KEY>` on the Synthetics page.
+4. **Devices**: Select which devices to run your check on. Available devices are `Laptop Large`, `Tablet`, and `Mobile Small`.
+5. **Locations**: Select which Datadog managed locations to run the test from. Many AWS locations from around the world are available. The full list is retrievable through the [Datadog API][5]. You can also set up a [Private Location][11] to run a Synthetics Browser test on a private URL not accessible from the public internet.
+6. **How often should Datadog run the test?** Set the interval for how often the test will run. Intervals are available between every 15 minutes to once per week. [Contact support][12] to enable additional frequencies for your test.
+7. **Alert conditions**: Set the conditions that need to be met (`minutes`, `locations`, and `retry`) for a test to retry if it has failed.
+8. **Notify your team**: Enter an message associated with the alert condition and who the alert should be sent to. Use `@` to tag people in the body of the message and select, from the dropdown list, who to send the alert to. You can also type in an email address in this location to send to any team or member that is not listed.
+9. Once you've input all required fields, click the *Save Details & Record Test* button.
+
+#### Record a test with variables and assertions
+
+Tests can be only recorded from **[Google Chrome][1]**. To record your test, download the [Datadog Record Test extension for Google Chrome][2].
+
+{{< img src="synthetics/browser_tests/browser_check_record_test.png" alt="Browser test record test" responsive="true" >}}
+
+1. Optionally, select **Open in a pop-up** at the upper right of the page to open your test recording in a separate pop-up window in order to avoid sizing issues in the displayed window within Datadog's interface.
```

Comment:
Would add:

Note: you can open a pop up in incognito mode by clicking `Open In A Pop Up` + `Shift`. This is particularly useful to record login workflows as the incognito mode will allow you to record your test without any cached browser data.